tests/rptest/tests/redpanda_oauth_test.py

```diff
@@ -66,35 +76,60 @@ def __init__(self,
             assert False, "Keycloak failed to start"
 
         security = SecurityConfig()
+        security.require_client_auth = True
```

Comment:
`require_client_auth` is for mTLS, which doesn't seem relevant here? I'm not opposed to it, just wondering if there's some side-effect I'm not aware of.

Reply:
>some side-effect

Don't think so - this is probably cruft from when I was thrashing about trying to diagnose CDT issues.